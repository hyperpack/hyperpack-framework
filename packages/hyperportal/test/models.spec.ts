import path from 'path'
import buildModels from '../src/models'

describe('models', () => {
  it('buildModels', async () => {
    const { models, connection } = await buildModels({
      appdir: path.join(__dirname, 'fixtures', 'type-orm-base')
    })
    expect(models).toMatchSnapshot()
    expect(connection).toBeTruthy()
  })
})
