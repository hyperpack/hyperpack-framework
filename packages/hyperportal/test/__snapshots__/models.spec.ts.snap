// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`models buildModels 1`] = `
Object {
  "PopBand": [Function],
}
`;
